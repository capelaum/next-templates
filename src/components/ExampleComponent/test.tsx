import { render, screen } from '@testing-library/react'

import { ExampleComponent } from '.'

describe('<ExampleComponent />', () => {
  it('should render the ExampleComponent heading', () => {
    const { container } = render(<ExampleComponent />)

    expect(
      screen.getByRole('heading', { name: /ExampleComponent/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
