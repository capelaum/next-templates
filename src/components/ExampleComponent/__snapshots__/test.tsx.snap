// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExampleComponent /> should render the ExampleComponent heading 1`] = `
<div
  class="wrapper"
>
  <h1>
    ExampleComponent
  </h1>
</div>
`;
